### ISSUE TITLE
False positive when strings are in `style` or `className` variables

### ISSUE THREAD

**Issue by tleunen** [sergioazoc/oxlint-tailwindcss#5]
These produce false positive even when the strings are not used for CSS.
```
const style = 'foobar'
const className = 'something'
```

tbh, I think it's "ok" to fail on `className`, but style can be really annoying at time

I noticed it in this context for example

<img width="1910" height="472" alt="Image" src="https://github.com/user-attachments/assets/c1785dbd-d07e-4ad1-869e-36e72689f2b8" />

**Comment by sergioazoc** [comment]
Fixed in v0.3.0.

You can now exclude specific items from the built-in defaults via `settings.tailwindcss.exclude`. To stop scanning variables named `style`/`styles`:

```jsonc
{
  "settings": {
    "tailwindcss": {
      "exclude": {
        "variablePatterns": ["^styles?$"]
      }
    }
  }
}
```

This removes the `/^styles?$/` pattern from the defaults while keeping `className`, `classNames`, and `classes` detection intact.

The `exclude` option also supports `attributes`, `callees`, and `tags` — so you can fine-tune exactly what the plugin scans:

```jsonc
{
  "settings": {
    "tailwindcss": {
      "exclude": {
        "variablePatterns": ["^styles?$"],
        "callees": ["objstr"]
      }
    }
  }
}
```
